src/Atmos/Model/get_prognostic_vars.jl

```diff
@@ -23,3 +23,6 @@ prognostic_vars(m::AtmosModel) = (
     prognostic_vars(m.tracers)...,
     prognostic_vars(m.turbconv)...,
 )
+
```

Comment:
I think we should be able to put `prognostic_vars(m::AtmosLinearModel) = ...` back in here, but include this file after `include("linear.jl")`. In fact, I think it can be the last include, because these methods are really only needed to auto-construct the tendency table.